src/client/al-api-client.ts

```diff
@@ -166,7 +182,9 @@ export class AlApiClient implements AlValidationSchemaProvider
     if ( cacheTTL && ! normalized.disableCache ) {
       let cachedValue = this.getCachedValue( fullUrl );
       if ( cachedValue ) {
-        this.log(`APIClient::XHR GET ${fullUrl} (from cache)` );
+        if ( this.verbose ) {
+          console.log(`APIClient::XHR GET ${fullUrl} (from cache)` );
```

Comment:
The `log` method on this class already checks to see if `this.verbose` is true -- why the change here?